install

```diff
@@ -60,8 +60,10 @@ download() {
 
 archi=$(uname -sm)
 case "$archi" in
-  Darwin\ x86_64) download skim-v$version-${binary_arch:-x86_64}-apple-darwin;;
-  Linux\ x86_64)  download skim-v$version-${binary_arch:-x86_64}-unknown-linux-musl;;
+  Darwin\ x86_64) download skim-${binary_arch:-x86_64}-apple-darwin;;
+  Linux\ x86_64)  download skim-${binary_arch:-x86_64}-unknown-linux-musl;;
+  Linux\ armv7l) download skim-${binary_arch:-armv7}-unknown-linux-musleabi;;
+  Linux\ aarch64) download skim-${binary_arch:-aarch64}-unknown-linux-musl;;
   *) ;;
```

Comment:
Could you please add a message for unknown architectures while we're changing this ?